docs/tsc/meetings/2019-07-22.md

```diff
@@ -0,0 +1,76 @@
+July 22, 2019
+
+Host: Michael Dolan
+
+Rotating Secretary: Michael Dolan
+
+Attendees:
+  * [X] Mark Boorer (_TSC_) - Industrial Light & Magic
+  * [ ] Sean Cooper (_TSC_) - DNEG
+  * [X] Michael Dolan (_TSC Chair_) - Sony Pictures Imageworks
+  * [X] Larry Gritz (_TSC_) - Sony Pictures Imageworks
+  * [X] Patrick Hodoul (_TSC_) - Autodesk
+  * [ ] John Mertic - Academy Software Foundation / Linux Foundation
+  * [ ] Carl Rand (_TSC_) - Foundry
+  * [X] Doug Walker (_TSC Chief Architect_) - Autodesk
+  * [X] Kevin Wheatley (_TSC_) - Framestore
+  * [X] Bernard Lefebvre - Autodesk
+
+# **OCIO TSC Meeting Notes**
+
+* Open Source Maintainers Meeting
+    - Michael: We want to invite ACES leads to the meeting at SIGGRAPH. I don't
+      have all their contact info. Asks if Doug can provide it?
+    - **TODO**: Doug does, and is willing to reach out to Annie and JZ. Later
+      (offline) asks Sean if he would like to, since he is our ACES TAC rep.
+    - Meeting is Wednesday at 4pm.
+    - Larry clarifies that the purpose of the meeting isn't to advertise ASWF
+      or even attract new projects, but to discuss the ASWFs role in the wider
+      ecosystem, and how it can better work with and help other projects.
+
+* SIGGRAPH Working Group Meeting
+    - Michael proposes this be more of a working group meeting than a TSC
+      meeting
+    - Doug: We have two hours, so the TSC portion could be 30 minutes, and use
+      the rest for working group.
+    - Doug: Those not at SIGGRAPH can connect via Zoom.
+    - **TODO**: Doug will send invite to both ocio-dev and OCIO v2 working group
+      (google group).
+    - Doug: We should discuss release/pre-release branch and/or tag naming
+        - Other topics are CLF write support and recent CPU renderer performance
+          work (PR #769).
+        - Will show OCIO v2 roadmap
+
+* SIGGRAPH OCIO BoF
+    - Patrick: We really don't have a full hour; more like 45-50 minutes.
+    - Doug: Timing will be important to allow for discussion. Let's add target
+      durations to the slide deck notes to help keep us on track.
+    - Patrick: Someone should monitor time during meeting to keep things moving
+    - Doug: in interest of time, AD will not do live demo. Demos can be done
+      following meeting on Doug's laptop if there's interest.
+    - Michael: I will keep general project updates brief.
+    - Mark: Can keep OpenColorMath talk under 10 minutes.
+    - Michael: Thomas won't be there, but will provide statement to talk about
+      ACES 1.1 config.
+        - His work is mostly done, but not checked in anywhere yet, so we'll
+          likely be pointing people at his fork. Probably Ok since it's still
+          being tested.
+        - Michael: We're still trying to talk to Alex about what to do with
+          OCIO-Configs repo.
+        - Work will be included in Nuke 12 beta.
+        - Doug: Planning to show OCIO ACES implementation in comparison with
+          Thomas' config.
```

Comment:
Doug: Will have a slide to talk about improved ACES Output Transform support we added.  Will have results vs. ACES 1.0.3 and Thomas' fork available on laptop for anyone interested.